src/simpleapi.jl

```diff
@@ -58,34 +58,62 @@ function list(ctx::KuberContext, O::Symbol, apiversion::Union{String,Nothing}=no
     end
 end
 
-function get(ctx::KuberContext, O::Symbol, name::String, apiversion::Union{String,Nothing}=nothing; kwargs...)
+function get(ctx::KuberContext, O::Symbol, name::String, apiversion::Union{String,Nothing}=nothing; num_tries::Integer=1, kwargs...)
     isempty(ctx.apis) && set_api_versions!(ctx)
 
     apictx = _get_apictx(ctx, O, apiversion)
     try
         apicall = eval(Symbol("read$O"))
-        return apicall(apictx, name; kwargs...)
+        @repeat num_tries try
+            return apicall(apictx, name; kwargs...)
+        catch e
+            @retry if isa(e, IOError)
+                @warn("Retrying ", "read$O")
```

Comment:
Automatic retries are better done silently IMO, unless someone is trying to debug. So maybe these should be `@debug` instead of `@warn`.

Reply:
resolved with 83094411864efa127273bec205bc3a4366cf49ce

I have added retry support for init stuff (also it is handled at JuliaRun.init as well) for `get` APIs.